### DISCUSSION TITLE
I'd love the progress component to have a prop to show the current value inside the progress bar

### DISCUSSION THREAD

**Discussion by bonface221** [chakra-ui/chakra-ui#8272] [Ideas]
### Description

I'd love to see the progress bar with the actual progres inside it.
![image](https://github.com/chakra-ui/chakra-ui/assets/98006412/ace45b58-ee67-4d76-a4a7-8755a98a0592)

Thanks chakra ui team

### Problem Statement/Justification

Nope

### Proposed Solution or API

![image](https://github.com/chakra-ui/chakra-ui/assets/98006412/9011f0b4-1b84-439e-aebb-99307412852a)

**Comment by kaushik1094** [comment]
Created a PR adding the feature : https://github.com/chakra-ui/chakra-ui/pull/8134 Please provide the feedback and can make necessary changes as needed
